Tests/Categories/NSBundle+TestBundle.h

```diff
@@ -0,0 +1,12 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NSBundle (TestBundle)
```

Comment:
Goal:
No Need for Host App bundling. We are able to access test bundle with helper category.

Change:
+ Add NSBundle category exposing TestBundle reference
+ Include Test files in this bundle which are accessible regardless of app target allowing for fewer target memberships